src/Components/Navbar.tsx

```diff
@@ -59,11 +59,9 @@ const Navbar = () => {
                   Categories
                 </button>
                 <ul className="dropdown-menu">
-                  <li><button className="dropdown-item" onClick={() => handleSelect("smartphones")}>smartphones</button></li>
-                  <li><button className="dropdown-item" onClick={() => handleSelect("laptops")}>laptops</button></li>
-                  <li><button className="dropdown-item" onClick={() => handleSelect("fragrances")}>fragrances</button></li>
-                  <li><button className="dropdown-item" onClick={() => handleSelect("groceries")}>groceries</button></li>
-                  <li><button className="dropdown-item" onClick={() => handleSelect("furniture")}>furniture</button></li>
+                  <li><button className="dropdown-item" onClick={() => handleSelect("Mobile")}>Mobile</button></li>
```

Comment:
This list should not be static. in fact nothing in UI should be static and come from db side

Reply:
I made changes, new the categories are dynamic coming from db.